09-secrets.md

```diff
@@ -1,28 +1,31 @@
 ## Secrets top-level element
 
-Secrets are a flavour of Configs focusing on sensitive data, with specific constraint for this usage. As the platform implementation may significantly differ from Configs, dedicated Secrets section allows to configure the related resources.
+Secrets provides a more secure way of getting sensitive information in to your application's services, so you don't have to rely on using environment variables. If you’re injecting passwords and API keys as environment variables, you risk unintentional information exposure. Environment variables are often available to all processes, and it can be difficult to track access. They can also be printed in logs when debugging errors without your knowledge. Using secrets mitigates these risks.
```

Comment:
The tone/wording here contrasts with the rest of the specification (`if you're`, `so you don't`). The rest of the Specification generally doesn't address the reader directly, and uses a more formal tone. I'm still not convinced this information should reside here, but if we want to include it I'd rather it be changed to match the rest of the Specification.